analytics/dev/django.Dockerfile

```diff
@@ -3,8 +3,17 @@ FROM python:3.10-slim
 # * psycopg2
 RUN apt-get update && \
     apt-get install --no-install-recommends --yes \
-    libpq-dev gcc libc6-dev git && \
-    rm -rf /var/lib/apt/lists/*
+    libpq-dev gcc libc6-dev git
+
+# Install spack dependencies
+RUN apt-get update && \
+    apt-get install --no-install-recommends --yes \
+    build-essential ca-certificates coreutils curl \
+    environment-modules gfortran git gpg lsb-release \
+    python3 python3-distutils python3-venv unzip zip
```

Comment:
I think you can switch the base image to just `python:3.10` instead of `python:3.10-slim`, and `python3-distutils`/`python3-venv` will be automatically installed.